README.md

```diff
@@ -163,6 +165,10 @@ Object caching is encouraged, as the plugin saves Sophi data as a transient.  If
 
 The default caching period is five minutes. This can be modified with the `sophi_cache_duration` hook.
 
+### Sophi API empty response
+
+If the Sophi API returns an empty Post ID array, the plugin will result in a "no results" response.  The `found_posts` filter can be used to modify the Sophi API response to allow manual posts to be injected into the response (e.g., via a fallback method to inject posts that would be a good fit).
```

Comment:
This would be the same filter `sophi_curated_post_list`. found_posts can be used as a reference for devs, this is actually an override from WP_Query. For example, if the response from Sophi is the empty array, found_posts would be 0 (and they can have more certainty we got an empty array)